packages/components/src/styles/__snapshots__/form-styles.spec.ts.snap

```diff
@@ -17,7 +17,7 @@ exports[`getBaseChildStyles() should return correct css for child: input, state:
     "margin": 0,
     "outline": 0,
     "textIndent": 0,
-    "transition": "color var(--p-transition-duration, .24s) ease,border-color var(--p-transition-duration, .24s) ease,background-color var(--p-transition-duration, .24s) ease",
+    "transition": "background-color var(--p-motion-duration, 0.25s) cubic-bezier(0.25,0.1,0.25,1), border-color var(--p-motion-duration, 0.25s) cubic-bezier(0.25,0.1,0.25,1), color var(--p-motion-duration, 0.25s) cubic-bezier(0.25,0.1,0.25,1)",
```

Comment:
Hmm, would have been less changes if the order would have stayed the same but doesn't matter now.